ydb/aio/table.py

```diff
@@ -342,7 +342,7 @@ async def acquire(self, timeout: float = None, retry_timeout: float = None, retr
 
         if self._should_stop.is_set():
             self._logger.debug("Acquired not inited session")
-            return self._create()
```

Comment:
session should stop or should initialize? I have readed code above and concluded that should_stop unset on init and there may be not initialize session (but uses didn't' want to stop it).